package.json

```diff
@@ -25,8 +25,7 @@
     "url": "https://github.com/button/eslint-config-button/issues"
   },
   "homepage": "https://github.com/button/eslint-config-button#readme",
-  "dependencies": {},
-  "peerDependencies": {
+  "devDependencies": {
     "eslint": ">=6.8.0",
```

Comment:
**change request**: I'm relatively confident that we don't want `devDependencies` for these. devDependencies are generally for tools that are only needed during the internal development lifecycle of a package, but all the packages we list here are needed in some way by the consumer. Stepping back a bit, if makes sense for us to not have any `devDependencies` here at all, since there's no test suite or build toolchain — `eslint-config-button` is just a JSON document 🙂 

Says ESLint:

> You should **declare your dependency on ESLint in package.json using the peerDependencies** field. The recommended way to declare a dependency for future proof compatibility is with the ">=" range syntax, using the lowest required ESLint version.
> 
> If your shareable config depends on **a plugin, you should also specify it as a peerDependency** (plugins will be loaded relative to the end user's project, so the end user is required to install the plugins they need). However, if your shareable config depends on **a third-party parser or another shareable config, you can specify these packages as dependencies**.
> 
> https://eslint.org/docs/developer-guide/shareable-configs#publishing-a-shareable-config

Based on my read of that: 
* `eslint`, and all of `eslint-plugin-*` should be `peerDependencies`
* `babel-eslint` (parser), `eslint-config-airbnb` (shareable config) should be `dependencies`

Reply:
Sounds good to me :) https://github.com/button/eslint-config-button/pull/11/commits/765a4252a8d0b5ff707e6acec2e8e9aa51ecc9d6